src/main/java/org/opentripplanner/middleware/bugsnag/BugsnagDispatcher.java

```diff
@@ -52,12 +52,9 @@
         "Content-Type", "application/json"
     );
 
-    /**
-     * Filter object defining the boundaries on which event requests will be based.
-     */
-    private static final String EVENT_REQUEST_FILTER = buildEventRequestFilter();
+    private static final Map<Integer, String> REQUEST_FILTERS = new HashMap<>();
```

Comment:
Please add JavaDoc about what this variable is used for and what the keys represent.

Reply:
After further examining the code, I don't think saving these request filters like this is needed since the computation and memory savings are extremely minimal.